src/ocean/util/serialize/contiguous/package_.d

```diff
@@ -696,3 +697,46 @@ unittest
     auto d = Deserializer.deserialize!(S2)(buffer);
     test(deepEquals(*d.ptr, s));
 }
+
+/******************************************************************************
+
+    Const arrays of arrays
+
+******************************************************************************/
+
+struct ConstS
+{
+    Const!(int) n = 3;
+    Const!(char[][]) a = ["Hello", "World"];
+    Const!(char[])[2][3] b = [
+        ["Die", "Katze"], ["tritt", "die"], ["Treppe", "krumm."]
+    ];
+}
+
+unittest
+{
+    void[] buffer;
+
+    ConstS s;
+
+    Serializer.serialize(s, buffer);
+    auto cont_S = Deserializer.deserialize!(ConstS)(buffer);
```

Comment:
As mentined in other issue, my intention was to not allow this, requiring target struct to be 100% mutable. Exactly because it simplifies (de)serializer code a lot while not being relevant to our typical use cases. What has caused you to pursue such struct support? Original plan was to go with using mutable structs internally and aplying `const` to them as a whole when accepting external params.